components/mission/MissionInfo.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div class="spacer"></div>
```

Comment:
This should be translated into a page instead of a component.
And each part of the page should be componentized, such as the comment section, description ...etc